sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,15 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  private[spark] val ADDITIONAL_REMOTE_REPOSITORIES =
+    ConfigBuilder("spark.sql.additionalRemoteRepositories")
+      .doc("A comma-delimited string config of the optional additional remote maven mirror " +
+        "repositories, this can be used for downloading Hive jars in IsolatedClientLoader " +
```

Comment:
`this can be used` -> `this is only used`?

I'm slightly worrying about that this might be confused with `--repositories`.

Reply:
Changed in 5bd630c.